CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [v0.1.76](https://github.com/magma-gpu/rutabaga_gfx/tree/v0.1.76)
+
+### API and dependencies
```

Comment:
I added only what I did.. please let me know if you want to add something, or you can add it..